Token_Contracts/test/humanStandardTokenFactory.js

```diff
@@ -11,14 +11,14 @@ contract("HumanStandardTokenFactory", function(accounts) {
 
         return factory.createHumanStandardToken.call(100000, "Simon Bucks", 2, "SBX", {from: accounts[0]});
         }).then(function(tokenContractAddr) {
-            console.log(tokenContractAddr);
             newTokenAddr = tokenContractAddr;
             return factory.createHumanStandardToken(100000, "Simon Bucks", 2, "SBX", {from: accounts[0]});
         }).then(function(result) {
-            console.log(result);
             return factory.verifyHumanStandardToken.call(newTokenAddr, {from: accounts[0]});
         }).then(function (result) {
-            console.log(result);
+            assert.strictEqual(result, true);
+            return factory.isHumanToken.call(newTokenAddr, {from: accounts[0]}); //use other check as well.
```

Comment:
I know what you mean, but I think the comment can be removed, or be rephrased, as it currently can sound like a "todo"...